packages/now-static-build/src/utils/read-build-output.ts

```diff
@@ -0,0 +1,164 @@
+  );
+  const staticFiles = await readStaticFiles({ workPath });
+
+  const routes = await readRoutesConfig({ workPath });
+
+  const outputs = {
+    staticFiles: isObjectEmpty(staticFiles) ? null : staticFiles,
+    functions: isObjectEmpty(functions) ? null : functions,
+    routes: routes.length ? routes : null,
+  };
+
+  if (outputs.functions) {
+    console.log(
+      `Detected Serverless Functions in ".vercel_build_output/functions"`
+    );
+  }
+
+  if (outputs.staticFiles) {
+    console.log(`Detected Static Assets in ".vercel_build_output/static"`);
+  }
+
+  if (outputs.routes) {
+    console.log(`Detected Configuration in ".vercel_build_output/config"`);
+  }
+
+  return outputs;
+}
+
+async function readStaticFiles({
+  workPath,
+}: {
+  workPath: string;
+}): Promise<Files> {
+  const staticFilePath = path.join(workPath, '.vercel_build_output', 'static');
+  const staticFiles = await glob('**', {
+    cwd: staticFilePath,
+  });
+
+  return staticFiles;
+}
+
+async function readNodeFunctions({
+  workPath,
+  functionsMountPath,
+}: {
+  workPath: string;
+  functionsMountPath: string;
+}) {
+  const output: { [key: string]: Lambda } = {};
+  const nodeFunctionPath = path.join(
+    workPath,
+    '.vercel_build_output',
+    'functions',
+    'node'
+  );
+  const nodeFunctionFiles = await glob('*/index.js', {
+    cwd: nodeFunctionPath,
+  });
+  const nodeBridgeData = await fs.readFile(path.join(__dirname, 'bridge.js'));
+
+  for (const fileName of Object.keys(nodeFunctionFiles)) {
+    const launcherFileName = '___now_launcher';
+    const bridgeFileName = '___now_bridge';
+
+    const launcherFiles: Files = {
+      [`${launcherFileName}.js`]: new FileBlob({
+        data: makeNowLauncher({
+          entrypointPath: `./index.js`,
+          bridgePath: `./${bridgeFileName}`,
+          helpersPath: '',
+          sourcemapSupportPath: '',
+          shouldAddHelpers: false,
+          shouldAddSourcemapSupport: false,
+        }),
+      }),
+      [`${bridgeFileName}.js`]: new FileBlob({
+        data: nodeBridgeData,
+      }),
+    };
+
+    const requiredFiles = await glob('**', {
+      cwd: path.join(nodeFunctionPath, path.dirname(fileName)),
+    });
+
+    const lambda = await createLambda({
+      files: {
+        ...requiredFiles,
+        ...launcherFiles,
+      },
+      handler: `${launcherFileName}.launcher`,
+      runtime: getLatestNodeVersion().runtime,
+    });
+
+    const parsed = path.parse(fileName);
+    const newPath = path.join(functionsMountPath, parsed.dir, parsed.name);
+    output[newPath] = lambda;
+
+    debug(
+      `Created Lambda "${newPath}" from "${path.join(
+        nodeFunctionPath,
+        fileName
+      )}".`
+    );
+  }
+
+  return output;
+}
+
+async function readRoutesConfig({
+  workPath,
+}: {
+  workPath: string;
+}): Promise<Route[]> {
+  const routesConfigPath = path.join(
+    workPath,
+    '.vercel_build_output',
+    'config',
```

Comment:
I think this should be `_config` or something that would be less likely for the user to use.

Imagine a user who makes a `config/index.js` function.

Reply:
@styfle this is the directory structure

```
.vercel_build_output
├── config
│   └── routes.json
├── functions
│   └── node
│       └── my-function
│           └── index.js
├── static
│   ├── index.html
│   └── about.html
```

They should either write to `config`, `functions`, or `static`, others will be ignored, so you have to explicitly put a file into functions to become a function.

Reply:
It is expected that frameworks (or whatever is doing the building), is creating files for `config` (at the moment only `routes.json`).